src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,227 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This test stress tests bucket unpacking. Each document is large with many top-level fields.
+  Each query targets a measurement, and cannot use the index on time/meta for time-series collections.
+  Each query will target a different number of buckets, and we should see the runtime for each query
+  increase based on the number of buckets unpacked.
+
+Keywords:
+  - timeseries
+  - aggregate
+
+GlobalDefaults:
+  Database: &database test
+  Collection: &collection Collection0
+  maxPhases: &maxPhases 10
+  Repeat: &repeat 100
+
+Actors:
+  # Clear any pre-existing collection state. This is helpful to avoid errors during local testing.
+  - Name: ClearCollection
+    Type: CrudActor
+    Database: *database
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Threads: 1
+          Collection: *collection
+          Operations:
+            - OperationName: drop
+
+  - Name: CreateTimeSeriesCollection
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *database
+          Operation:
+            OperationMetricsName: CreateTimeSeriesCollection
+            OperationName: RunCommand
+            OperationCommand:
+              { create: *collection, timeseries: { timeField: "time" } }
+
+  - Name: InsertData
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [2]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *database
+          CollectionCount: 1
+          Threads: 1
+          DocumentCount: 1000000
+          BatchSize: 1000
+          Document:
+            # Since step = 900 ms and documentCount = 1,000,000, the first and last document
+            # would be 900,000,000 ms apart (roughly 10 days).
+            time: { ^IncDate: { start: "2022-01-01", step: 900 } }
```

Comment:
This means there will be no "out-of-order" events, making for very neat buckets. Maybe add a comment why this is OK/desireable for this workload?